autofs-5.1.3 - make map source reference message debug only

From: Ian Kent <raven@themaw.net>

The message "map source used without taking reference" was added during
development of an additional amd format map feature.

It appears far too often but hasn't been seen to be a real problem so
make it a debug only message and postpone deeper investigation until
later.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG    |    1 +
 lib/master.c |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -262,6 +262,7 @@
 - make open_lookup() error handling more consistent.
 - be silent about sss library not found.
 - be silent about nis domain not set.
+- make map source reference message debug only.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/lib/master.c
+++ autofs-5.0.7/lib/master.c
@@ -233,7 +233,7 @@ master_add_map_source(struct master_mape
 
 		this = __master_find_map_source(entry, type, format, argc, tmpargv);
 		if (this) {
-			error(entry->ap->logopt,
+			debug(entry->ap->logopt,
 			      "map source used without taking reference");
 			this->age = age;
 			master_free_map_source(source, 0);
